tests/io/test_read_commandline.py

```diff
@@ -0,0 +1,48 @@
+import os
+import pandas as pd
+import janitor.io
+
+
+"""
+Tests seek to assert that a dataframe made
+    from the read_commandline function
+    will create a dataframe identical
+    to a test dataframe
+
+Tests 1 and 2 assert are admittedly trivial,
+    they effectively prove the validity of
+    the pd.testing library by proving the
+    test data is equal to itself. Note the
+    requirement of 'assert not' for
+    assert_frame_equal.
+
+Tests 3 and 4 assert that the dataframe made
+    from the read_commandline function is
+    identical to the test dataframe from
+    which the .csv file was created.
+
+"""
```

Comment:
This should go into the test function as a docstring, rather than as a string outside of the function. 

Additionally, I think it can be shortened. It seems like only tests 3 and 4 are necessary.